api/descriptions/collections/post.md

```diff
@@ -23,5 +23,6 @@ $ curl 'https://<CONSOLE>/api/v<VERSION>/collections' \
 }'
 ```
 Any resource left unspecified, such as `hosts`, `functions`, or `clusters`, is assigned a wildcard by default.
+Resources that are not explicitly specified need to have a `*` wildcard applied to them in order to instantiate.
+
 
-**Note:** No response will be returned upon successful execution.
```

Comment:
This is still valid. Not sure, why this got removed.

Reply:
There are situations where we receive no response due to perhaps a 404/400 error but the cURL has no response shown, and this indicates that it is successful. However, it is a failure. Running cURL -vvv on your API calls can show this in some aspects.

Reply:
If you are certain that this functionality is behaving as expected, perhaps it is some bug with cURL or some other variable, and I can research that and see what I find and create a different PR/Report for that aspect.

Reply:
True, but we have several other instances similar to this. Shall we remove such statements from other API descriptions too?

Reply:
I personally believe we should. I do not think there is any added value from having this statement, as it can confuse customers. It also is a blanket statement that doesn't necessarily cover every single use-case. Also, product team (I assume) is not keeping up with that as some sort of standard. What I do know, is that I have seen this occur where we have no response, but it's not completing the API call. It's getting a 404. I think that we get more benefit from removing this statement, than we do from having it in the docs. Generally, customers using the API are well-off enough to navigate around themselves and will know whether or not it works.

Reply:
@ssugandh What is the response code when a collection is successfully created? 200?

Reply:
@iansk : There's no response for a successful execution. I have retained the previous line with a caveat to verify with the Console UI.